pandas/_libs/testing.pyx

```diff
@@ -73,31 +68,33 @@ cpdef assert_almost_equal(a, b,
     ----------
     a : object
     b : object
-    check_less_precise : bool or int, default False
-        Specify comparison precision.
-        5 digits (False) or 3 digits (True) after decimal points are
-        compared. If an integer, then this will be the number of decimal
-        points to compare
+    rtol : float, default 1e-5
+        Relative tolerance.
+
+        .. versionadded:: 1.1.0
+    atol : float, default 1e-8
```

Comment:
I *think* that you need a blank line after the versionadded to avoid a docs error

Reply:
Hmmm I copied from other parts of the code, and tested it with Sphinx.

Just ran `cd doc && python make.py html --num-jobs 4` and the docs look good :+1: